pkg/models/media.go

```diff
@@ -23,33 +23,50 @@ type MediaStorer interface {
 	AddLogoPathDb(path string,userId int)
 	GetLogo(userId int) string
 	GetExistedLogo(userId int) []string
-	//AddMediaPathDb(path string,userId int)
+	AddMediaPathDb(path string,userId int)
+	GetExistedGallery(userId int) []string
 }
 
 func (m *MediaStore) AddLogoPathDb(path string,userId int){
-	_, err := m.DB.Exec("insert into media (logo_path,user_id)VALUES ($1,$2)", path, userId)
+	_, err := m.DB.Exec("insert into logos (logo_path,user_id)VALUES ($1,$2)", path, userId)
+	if err != nil {
+		log.Println(err)
```

Comment:
We have package 'logger', use it. Do not log errors in models, handle them in controllers. Return errors from model if they occur.